### PR TITLE
Update enable-network-protection.md

### PR DESCRIPTION
**Pull request by msbemba**
added statement to update  Microsoft Defender antimalware platform before block or audit.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7856

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/enable-network-protection.md [hunk 1]
```diff
@@ -85,6 +86,9 @@ Enable network protection by using any of these methods:
 
 Use the [./Vendor/MSFT/Policy/Config/Defender/EnableNetworkProtection](/windows/client-management/mdm/policy-csp-defender) configuration service provider (CSP) to enable or disable network protection or enable audit mode.
 
+[Update Microsoft Defender antimalware platform]( to the latest version](https://support.microsoft.com/en-us/topic/update-for-microsoft-defender-antimalware-platform-92e21611-8cf1-8e0e-56d6-561a07d144cc) before you enable or disable network protection or enable audit mode.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-network-protection.md]:
```suggestion
[Update Microsoft Defender antimalware platform to the latest version](https://support.microsoft.com/topic/update-for-microsoft-defender-antimalware-platform-92e21611-8cf1-8e0e-56d6-561a07d144cc) before you enable or disable network protection or enable audit mode.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba @JohanFreelancer9

**Pull request merged**